risc0/zkvm/platform/src/syscall.rs

```diff
@@ -71,6 +71,9 @@ pub mod reg_abi {
 
 pub const DIGEST_WORDS: usize = 8;
 pub const DIGEST_BYTES: usize = WORD_SIZE * DIGEST_WORDS;
+pub const MAX_BUF_BYTES: usize = 64 * 1024;
+pub const MAX_BUF_WORDS: usize = MAX_BUF_BYTES / WORD_SIZE;
+pub const MAX_SHA_COMPRESS_ROUNDS: usize = 1000;
```

Comment:
Do we also need to add a check to the build for the `Executor` that the segment limit is not set low enough that the max number of read or compress invocations does not exceed the segment limit?

Reply:
Good call, I'll review what the new minimum needs to be based on these parameters.